bowlingGame.js

```diff
@@ -0,0 +1,29 @@
+// class BowlingGame
+
+//     def initialize
+//         @rolls = []
+//     end
+
+//     def roll pins 
+//         @rolls.push pins
+//     end
+
+//     def score
+//         result = 0
+//         rollIndex = 0
+//         10.times do
+//           if @rolls[rollIndex] + @rolls[rollIndex + 1] == 10
+//             result += @rolls[rollIndex] + @rolls[rollIndex + 1] = @rolls[rollIndex + 2]
+//           else
+//             result += @rolls[rollIndex] + @rolls[rollIndex + 1]
+//           end
+//           rollIndex += 2
+//         end
+//         result
+//     end
+
+//     def spare? rollIndex
+
+//     end
+
```

Comment:
Nice work, you've got the strike calculation and normal frame score calculation. What you could add is the spare score calc, which is the same as the strike score code, except when you come to advance the rollIndex, you advance by 2 (rollIndex += 2 when it's a normal frame or spare, rollIndex +=1 when it's a strike).